frontend/dispatcher.go

```diff
@@ -17,6 +17,47 @@ type HTTPError struct {
 	Message string
 }
 
+func db_requester() {
+	for {
+		db_request := <-db_open_channel
```

Comment:
@Xe Sure but why is it a good idea here? Why am I adding a `select` statement to something that is just one thing? I'll never have to introspect whatever is coming over the wire, right?